.github/release-drafter.yml

```diff
@@ -1,5 +1,5 @@
-name-template: "$NEXT_PATCH_VERSION"
-tag-template: "v$NEXT_PATCH_VERSION"
+name-template: "$RESOLVED_VERSION"
+tag-template: "v$RESOLVED_VERSION"
```

Comment:
🗒️ This tells Release Drafter to intelligently pick the next version number, rather than always doing a patch bump.